pkg/cluster/member.go

```diff
@@ -38,6 +39,11 @@ func (c *Cluster) updateMembers(known etcdutil.MemberSet) error {
 				return errUnexpectedUnreadyMember
 			}
 
+			if !strings.HasPrefix(m.Name, c.cluster.Metadata.GetName()) {
```

Comment:
This is not right.
The non-managed member could have a name not prefixed with cluster name.